docker-compose.yml

```diff
@@ -1,17 +1,26 @@
 version: '3.9'
 
-x-common: &common
+x-frontend: &frontend
   environment:
-    VITE_USER_POOLS_ID: ca-central-1_t2HSZBHur
+    VITE_USER_POOLS_ID: ca-central-1_FxagSNQa7
```

Comment:
question (blocking): is this change expected? I believe the previous vaue is the right one for DEV.

Reply:
Thanks for the correction!